products/chrome.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Chrome
+category: app
+tags:
+  - google
+  - web-browser
+iconSlug: googlechrome
+permalink: /chrome
+alternate_urls:
+-   /google-chrome
+versionCommand: google-chrome --version
+releasePolicyLink: https://blog.chromium.org/2021/03/speeding-up-release-cycle.html
```

Comment:
```suggestion
releasePolicyLink: https://developer.chrome.com/docs/web-platform/chrome-release-channels
```

The https://blog.chromium.org/2021/03/speeding-up-release-cycle.html may be used in the description, with a sentence such as : [Since 2021](https://blog.chromium.org/2021/03/speeding-up-release-cycle.html), Chrome is released every 4 weeks.